js/doc.js

```diff
@@ -98,11 +98,12 @@ function CRD() {
 }
 
 function SchemaPart({ key, property, parent, parentSlug }) {
-  const [props, propKeys, required, type, schema] = useMemo(() => {
+  const [props, propKeys, required, type, schema, enumvals] = useMemo(() => {
     let schema = property;
     let props = property.Properties || {};
 
     let type = property.Type;
+    let enumvals = null;
```

Comment:
Why not return an empty array instead of null, when the property is not an enum? See also below.